testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -167,6 +168,39 @@ public void testSelectOnViewFromDifferentSchema()
         assertViewQuery("SELECT * FROM test_schema.hive_test_view_1", queryAssert -> queryAssert.hasRowsCount(25));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testViewReferencingTableInDifferentSchema()
+    {
+        String schemaX = "test_view_table_in_different_schema_x" + randomTableSuffix();
+        String schemaY = "test_view_table_in_different_schema_y" + randomTableSuffix();
+        String tableName = "test_table";
+        String viewName = "test_view";
+
+        onHive().executeQuery(format("CREATE SCHEMA %s", schemaX));
+        onHive().executeQuery(format("CREATE SCHEMA %s", schemaY));
+
+        onTrino().executeQuery(format("CREATE TABLE %s.%s AS SELECT * FROM tpch.tiny.nation", schemaY, tableName));
+        onHive().executeQuery(format("CREATE VIEW %s.%s AS SELECT * FROM %s.%s", schemaX, viewName, schemaY, tableName));
+
+        assertThat(onTrino().executeQuery(format("SELECT COUNT(*) FROM %s.%s", schemaX, viewName))).containsOnly(row(25));
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testViewReferencingTableInTheSameSchemaWithoutQualifier()
+    {
+        String schemaX = "test_view_table_same_schema_without_qualifier_schema" + randomTableSuffix();
+        String tableName = "test_table";
+        String viewName = "test_view";
+
+        onHive().executeQuery(format("CREATE SCHEMA %s", schemaX));
+
+        onTrino().executeQuery(format("CREATE TABLE %s.%s AS SELECT * FROM tpch.tiny.nation", schemaX, tableName));
+        onHive().executeQuery(format("USE %s", schemaX));
```

Comment:
If the `USE schema` statement is not used the test probably would run just a bit more faster with the same results (`CREATE VIEW` statement would probably need to be adjusted with the schema name as well).

Reply:
yes, but the point of this test is to create view from table in the same scheme without qualifier, I hoped test name is clear about that :/